org.eclipse.jdt.ls.debug/src/org/eclipse/jdt/ls/debug/adapter/jdt/JDTProviderContext.java

```diff
@@ -0,0 +1,26 @@
+package org.eclipse.jdt.ls.debug.adapter.jdt;
+
+import org.eclipse.jdt.ls.debug.adapter.IProviderContext;
+import org.eclipse.jdt.ls.debug.adapter.ISourceLookUpProvider;
+import org.eclipse.jdt.ls.debug.adapter.IVirtualMachineManagerProvider;
+
+public class JDTProviderContext implements IProviderContext {
```

Comment:
The convention is to only capitalize the first letter. So JDTProviderContext => JdtProviderContext